### PR TITLE
Remove unused events and join templates

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary
Removes unused out-dated templates that are not needed for the migration.

|template|function|status|
|--------------------------------|---------------------|---------------------------|
|stage-youtube-live.html|display future events|remove|
|stage-adobe_connect.html|display future events|remove (not currently used)|
|community-join-slack.html|join event by slack|remove (not used)|
|community-join-google-group.html|join event by google|remove (not used)|
|community-join-without-form.html|join event by email|remove (not used)|

### CODE REVIEW

**Comment by RileySeaburg** [comment]
@ToniBonittoGSA, I'll let you approve & merge this

**Comment by ToniBonittoGSA** [comment]
> @ToniBonittoGSA, I'll let you approve & merge this

I left a noite in JIRA, @RileySeaburg

**Comment by nick-mon1** [comment]
@ToniBonittoGSA @RileySeaburg 
This PR is just for removing un-used templates. #8138 addresses the last mod date.

File: themes/digital.gov/layouts/partials/core/last-updated.html
Comment (RileySeaburg) [inline themes/digital.gov/layouts/partials/core/last-updated.html:1]: @nick-mon1 this introduces an additional file no?

I believe this is what we want removed first.

**Review by RileySeaburg — CHANGES_REQUESTED** [review]

File: themes/digital.gov/layouts/partials/core/last-updated.html
Comment (nick-mon1) [reply themes/digital.gov/layouts/partials/core/last-updated.html:1]: Ahhh, this should not be here. Thanks

**Pull request closed without merging**